build/config/android/config.gni

```diff
@@ -43,7 +43,7 @@ if (is_android || is_chromeos) {
 
     # The default to use for android:minSdkVersion for targets that do
     # not explicitly set it.
-    default_min_sdk_version = 24
+    default_min_sdk_version = 26
```

Comment:
I suppose I could make a new gni file or new variable in this file that gets used by these 3 variables?

Reply:
In theory we could also later try and change our Starboard code to make it build with Chromium 24 target, but for now it's fine to just leave it as is. 

We do need to actively come back and re-assess diffs like this though.

Reply:
Added b/374305510